packages/gitbeaker-core/src/services/ContainerRegistry.ts

```diff
@@ -32,13 +32,15 @@ export class ContainerRegistry extends BaseService {
   removeTags(
     projectId: string | number,
     repositoryId: number,
-    tagNameRegex: string,
-    options?: Sudo & { keepN: string; olderThan: string },
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    name_regex: string,
+    options?: Sudo & { keep_n: string; older_than: string },
   ) {
     const [pId, rId] = [projectId, repositoryId].map(encodeURIComponent);
 
     return RequestHelper.del(this, `projects/${pId}/registry/repositories/${rId}/tags`, {
-      tagNameRegex,
+      // eslint-disable-next-line @typescript-eslint/camelcase
+      name_regex,
```

Comment:
Please remove the disable of lint
```suggestion
      nameRegex,
```

is another way to solve it:
```suggestion
      nameRegex: tagNameRegex,
```

Reply:
The first would be cleaner, just changing the name of the variable